main/menus.js

```diff
@@ -45,14 +46,16 @@ const openFileItem = {
   click: async () => {
     closeAllCroppers();
 
-    const {canceled, filePaths} = await dialog.showOpenDialog({
-      filters: [{name: 'Videos', extensions: supportedVideoExtensions}],
-      properties: ['openFile', 'multiSelections']
-    });
+    ensureDockIsShowing(async () => {
```

Comment:
```suggestion
    await ensureDockIsShowing(async () => {
```

It doesn't matter in this case, but if you add something after this block, it could be a problem.